latest/ug/automode/auto-odcr.adoc

```diff
@@ -0,0 +1,34 @@
+include::../attributes.txt[]
+
+[.topic]
+[#auto-odcr]
+= Control deployment of workloads into EC2 On-Demand Capacity Reservations with EKS Auto Mode
+:info_titleabbrev: Control ODCR Deployment
+
+EC2 On-Demand Capacity Reservations (ODCRs) allow you to reserve compute capacity for your Amazon EC2 instances in a specific Availability Zone for any duration. When using EKS Auto Mode, you may want to control whether your Kubernetes workloads are deployed onto these reserved instances to maximize utilization of pre-purchased capacity or to ensure critical workloads have access to guaranteed resources.
+
+By default, EKS Auto Mode automatically launches into open ODCRs. However, by configuring `capacityReservationSelectorTerms` on a NodeClass, you can explicitly control which ODCRs your workloads use. Once this feature is enabled, EKS Auto Mode will no longer automatically use open ODCRs—they must be explicitly selected by a NodeClass, giving you precise control over capacity reservation usage across your cluster.
+
+[WARNING]
+====
+If you configure `capacityReservationSelectorTerms` on a NodeClass in a cluster, EKS Auto Mode will no longer automatically use open ODCRs for _any_ NodeClass in the cluster.
+====
+
+== Example NodeClass
+
+```yaml
+apiVersion: eks.amazonaws.com/v1
+kind: NodeClass
+spec:
+  # Optional: Selects upon on-demand capacity reservations and capacity blocks
+  # for EKS Auto Mode to prioritize.
+  capacityReservationSelectorTerms:
+    - id: cr-56fac701cc1951b03
+    # Alternative Approaches
+    - tags:
+        Name: "targeted-odcr"
```

Comment:
I know I suggested this, but I've had second thoughts. Typically selection will be on a group, not an individual ODCR, and I don't want folks to think we only support targeted. Maybe something like this?
```suggestion
        app: "my-app"
```